source/elements/oneDAL/source/bibliography.rst

```diff
@@ -4,4 +4,14 @@
 Bibliography
 ============
 
-For more information about algorithms implemented in |dal_short_name|, refer to the following publications:
+For more information about algorithms implemented in |dal_full_name| (|dal_short_name|), refer to the following publications:
+
+.. [OpenCLSpec]
```

Comment:
it seems to be formatting issue, I see two tables, each contains one reference, and the columns of the tables are not aligned

Reply:
you are viewing it here on github, but github doesn't handle rst files very well
everything's fine in the ouput

Reply:
how to get the output?

Reply:
I'm building it on my local machine, but maybe @napetrov can crate a gh-pages branch for his fork and put htmls there?

Reply:
if you need instructions on how to build it locally:
- clone the fork
- go to `source/elements/oneDAL`
- run `sphinx-build source build` from cmd
- the output is in the `build` folder

you'd need python, sphinx and some sphinx extensions installed

Reply:
yes, will do now and will post link 
also you can grab tar archive from CI.

Reply:
the link will help

Reply:
https://napetrov.github.io/oneapi-spec/ 
this is current version generated here.  
working on comments and updates.

Reply:
it looks good, in terminology section, in dataset def we need to highlight 'responses'

Reply:
DPC++ , we probably also need a full name

Reply:
do we have the official Intel def of DPC?

Reply:
minor: in bibliography section the link to pdf seems to be inactive

Reply:
The current version of oneDAL provides Data Parallel C++ (DPC++) API extensions to the traditional C++ interface.

Is this still truth?

Reply:
datasets definition updated with cross slink to responses
DPC++ used with full name
I've rewrite it using official definition 
Yes, current version of oneDAL provides both interfaces

Reply:
As for link in bibliography @outoftardis  - might be you can help here? Fisrt link is indeed inactive and i can't fix this.

Reply:
@napetrov syntax like this works:

```
`some text <url>`_
```

Reply:
thanks. fixed.